contrib/sssd.spec.in

```diff
@@ -36,6 +36,12 @@
 %global build_passkey 0
 %endif
 
+%if 0%{?rhel} >= 10
```

Comment:
You should also add Fedora40+

Reply:
I talked to Alexey and he said we can add the fedora condition later, as it does not make sense until we build fedora with `--with-sssd-user=sssd`

Reply:
In that case I'm fine with it